scripts/globals/mobskills/super_buff.lua

```diff
@@ -0,0 +1,19 @@
+---------------------------------------------
+-- Spike Flail
+-- Deals extreme damage in a threefold attack to targets behind the user.
```

Comment:
Be sure to change the copy-paste comment block 😉

Reply:
Teo beat me to my comment again. ☹️

Reply:
jinx